managed/models/settings_helpers.go

```diff
@@ -226,9 +226,8 @@ func UpdateSettings(q reform.DBTX, params *ChangeSettingsParams) (*Settings, err
 		settings.DefaultRoleID = *params.DefaultRoleID
 	}
 
-	if len(params.EncryptedItems) != 0 {
-		settings.EncryptedItems = params.EncryptedItems
-	}
+	settings.EncryptedItems = params.EncryptedItems
```

Comment:
we need to update this value only if it's passed, so we need to somehow differentiate empty slice from value not being passed. Should we compare it with nil?